lazy_tensor_core/dispatch_lazy_mode.py

```diff
@@ -0,0 +1,54 @@
+import torch
+
+import lazy_tensor_core
+lazy_tensor_core._LAZYC._ltc_init_ts_backend()
+import lazy_tensor_core.debug.metrics as metrics
+import lazy_tensor_core.core.lazy_model as ltm
+
+torch._C._jit_set_nvfuser_enabled(True)
+torch._C._jit_set_texpr_fuser_enabled(False)
+torch._C._jit_set_profiling_executor(True)
+torch._C._jit_set_profiling_mode(True)
+torch._C._jit_override_can_fuse_on_cpu(False)
+torch._C._jit_override_can_fuse_on_gpu(False)
+torch._C._jit_set_bailout_depth(20)
+
+class lazy_execute(object):
+    def __init__(self, enabled=True):
+        self._enabled = enabled
+        
+    def __enter__(self):
+        lazy_tensor_core._LAZYC._ltc_enable_lazy_mode()
+               
+    def __exit__(self, type, value, traceback) : 
+        lazy_tensor_core._LAZYC._ltc_sync_live_tensors(lazy_tensor_core._LAZYC._ltc_get_default_device(), [], False)
```

Comment:
hmm, why do sync_live_tensors, then mark_step?  isn't mark_step a super-set of sync_live_tensors?

Reply:
This is just me not knowing what I'm doing :)
Thanks for pointing this out.